### DISCUSSION TITLE
Does anyone have any experience with the  Litagurizer script?

### DISCUSSION THREAD

**Discussion by pa-0** [ryanoasis/nerd-fonts#1755] [Q&A] [answered]
I am currently working on patching a font with nerd-font glyphs and would also like to add litagures.  

Does anyone have any experience with [Ligaturizer](https://github.com/ToxicFrog/Ligaturizer), spexifically with respect to the order in which the fonts should be patched with nerd fonts and litagures.  

Does it matter which I add to the font first?

**Comment by Finii** [comment, marked as answer]
Me: No experience.

I would first add the ligatures. Per default it seems to add something to the font's name, which can be problematic when also "Nerd Font ..." is added later on; I would probably use their no-rename switch. `font-patcher` needs to be run with the appropriate `--makegroups`. 🤷‍♀️ 

But in principle the order should not matter, unless they put the ligature glyphs into PUA. Probably run `font-patcher` on a ligature font with `--careful` and check for skipping glyph messages. (Well, the order still does not matter much; it would be a hard codepoint conflict that needs to be solved manually.)

**Reply by pa-0** [reply]
I appreciate the input, @Finii -- you've been a serious help these past few days!  I'm sure you're busy enough as it is, so I'll keep any follow-up ?'s to a minimum.

**Comment by pa-0** [comment]
### Reddit Mono (Light) Patched with nerd-font glyphs and litagures (from Fira Code)

![image](https://github.com/user-attachments/assets/888a4cfd-cab6-4302-bbc3-b3bb194a6df1)
